packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -66,6 +71,27 @@ export default {
       onHidden: () => this.$refs.menubutton.$el.focus(),
       onShown: () => this.$refs.menu.$el.querySelector('a:first-of-type').focus()
     })
+  },
+  methods: {
+    async clickApp(appEntry) {
+      // @TODO use id or similar
+      if (appEntry.iconMaterial === 'switch_ui') {
+        await this.setClassicUIDefault()
+      }
+    },
+    setClassicUIDefault() {
+      const endpoint = new URL(this.configuration.server || window.location.origin)
+      endpoint.pathname =
+        endpoint.pathname.replace(/\/$/, '') + '/index.php/apps/web/settings/default'
+      const headers = new Headers()
```

Comment:
instead of using fetch you could use `clientService.httpAuthenticated` which re-uses an axios instance being cached for the current access token.